tasks/test-project/util.js

```diff
@@ -16,12 +16,17 @@ async function applyCodemod(codemod, target) {
   await execa('yarn transform', args, getExecaOptions(path.resolve(__dirname)))
 }
 
+const RW_FRAMEWORKPATH = path.join(__dirname, '../../')
```

Comment:
```suggestion
const RW_FRAMEWORK_PATH = path.join(__dirname, '../../')
```

I find this more readable, but others can decide.